src/Parser/LASLoader.js

```diff
@@ -71,13 +71,16 @@ class LASLoader {
         */
         const scanAngles = new Float32Array(view.pointCount);
 
+        // For precision we take the first point that will be use as origin for a local referentiel.
+        const origin = getPosition.map(f => f(0)).map(val => Math.floor(val));
```

Comment:
Why is it necessary to floor the value ?

Reply:
To have integer instead of float. With float you can have an extra precision issus with such high numbers.